### PR TITLE
grpc api was finished

### PR DESCRIPTION
**Pull request by carlosfrey-encora**

### CODE REVIEW

File: cmd/grpc/client/main.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+const (
+	defaultName = "world"
+)
+
+var (
+	addr = flag.String("addr", "localhost:50051", "the address to connect to")
```
Comment (eoliveira-daitan) [inline cmd/grpc/client/main.go:19]: Quick tip for yah: How about make the address an env var?

like, one for the host... something like:

```
GRPC_HOST=localhost
```

And a second one for the port:

```
GRPC_PORT=50051
```

File: cmd/grpc/server/main.go [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+	pb "github.com/carlosfrey-encora/oficial-rampup/contract"
+	repository "github.com/carlosfrey-encora/oficial-rampup/internal/repository"
+	"github.com/go-sql-driver/mysql"
+	"google.golang.org/grpc"
+	gormdriver "gorm.io/driver/mysql"
+	gormmysql "gorm.io/gorm"
+)
+
+var (
+	port = flag.Int("port", 50051, "The server port")
```
Comment (eoliveira-daitan) [inline cmd/grpc/server/main.go:21]: Same here, how about use this port from an env var like `GRPC_PORT`?

**Review by eoliveira-daitan — CHANGES_REQUESTED** [review]

**Review by eoliveira-daitan — APPROVED** [review]

File: cmd/grpc/server/main.go (hunk 2 above)
Comment (carlosfrey-encora) [reply cmd/grpc/server/main.go]: done

File: cmd/grpc/client/main.go (hunk 1 above)
Comment (carlosfrey-encora) [reply cmd/grpc/client/main.go]: done

**Pull request merged**